# Replaces the inox 0005-disable-default-extensions.patch
# The difference being that this patch removes the CloudPrint, Feedback and InAppPayment 
# like the inox patch but leaves the Webstore in so we can access and download extensions.

diff '--color=auto' -ruN a/chrome/browser/extensions/component_extensions_whitelist/whitelist.cc b/chrome/browser/extensions/component_extensions_whitelist/whitelist.cc
--- a/chrome/browser/extensions/component_extensions_whitelist/whitelist.cc	2019-08-09 08:47:33.000000000 -0600
+++ b/chrome/browser/extensions/component_extensions_whitelist/whitelist.cc	2019-08-20 13:18:09.347250844 -0600
@@ -25,7 +25,6 @@
 
 bool IsComponentExtensionWhitelisted(const std::string& extension_id) {
   const char* const kAllowed[] = {
-    extension_misc::kInAppPaymentsSupportAppId,
     extension_misc::kMediaRouterStableExtensionId,
     extension_misc::kPdfExtensionId,
 #if defined(OS_CHROMEOS)
diff '--color=auto' -ruN a/chrome/browser/extensions/component_loader.cc b/chrome/browser/extensions/component_loader.cc
--- a/chrome/browser/extensions/component_loader.cc	2019-08-09 08:47:33.000000000 -0600
+++ b/chrome/browser/extensions/component_loader.cc	2019-08-20 13:23:04.277909016 -0600
@@ -448,11 +448,6 @@
   AddKeyboardApp();
 #else  // defined(OS_CHROMEOS)
   DCHECK(!skip_session_components);
-#if BUILDFLAG(ENABLE_PRINTING)
-  // Cloud Print component app. Not required on Chrome OS.
-  Add(IDR_CLOUDPRINT_MANIFEST,
-      base::FilePath(FILE_PATH_LITERAL("cloud_print")));
-#endif  // BUILDFLAG(ENABLE_PRINTING)
 #endif  // defined(OS_CHROMEOS)
 
   if (!skip_session_components) {
@@ -537,12 +532,6 @@
 #if BUILDFLAG(ENABLE_HANGOUT_SERVICES_EXTENSION)
     AddHangoutServicesExtension();
 #endif  // BUILDFLAG(ENABLE_HANGOUT_SERVICES_EXTENSION)
-    bool install_feedback = enable_background_extensions_during_testing;
-#if defined(GOOGLE_CHROME_BUILD)
-    install_feedback = true;
-#endif  // defined(GOOGLE_CHROME_BUILD)
-    if (install_feedback)
-      Add(IDR_FEEDBACK_MANIFEST, base::FilePath(FILE_PATH_LITERAL("feedback")));
   }
 
 #if defined(OS_CHROMEOS)
diff '--color=auto' -ruN a/chrome/browser/extensions/external_component_loader.cc b/chrome/browser/extensions/external_component_loader.cc
--- a/chrome/browser/extensions/external_component_loader.cc	2019-08-09 08:47:33.000000000 -0600
+++ b/chrome/browser/extensions/external_component_loader.cc	2019-08-20 13:24:01.445338787 -0600
@@ -29,9 +29,6 @@
 
 void ExternalComponentLoader::StartLoading() {
   auto prefs = std::make_unique<base::DictionaryValue>();
-#if defined(GOOGLE_CHROME_BUILD)
-  AddExternalExtension(extension_misc::kInAppPaymentsSupportAppId, prefs.get());
-#endif  // defined(GOOGLE_CHROME_BUILD)
 
 #if defined(OS_CHROMEOS)
   {
